ci-operator/step-registry/janus-idp/backstage-showcase/nightly/janus-idp-backstage-showcase-nightly-commands.sh

```diff
@@ -0,0 +1,18 @@
+#!/bin/bash
+export HOME WORKSPACE
+HOME=/tmp
+WORKSPACE=$(pwd)
+cd /tmp || exit
+
+export GITHUB_ORG_NAME GITHUB_REPOSITORY_NAME NAME_SPACE TAG_NAME
+
+GITHUB_ORG_NAME="janus-idp"
+GITHUB_REPOSITORY_NAME="backstage-showcase"
+NAME_SPACE=showcase-ci
```

Comment:
This namespace might conflict with the PR check.
Can we update it to `showcase-ci-nightly`? wdyt?